chalicelib/app.py

```diff
@@ -4,14 +4,33 @@
 from chalicelib.app_utils import AppUtils
 from chalicelib.deploy import Deploy
 
-
 # Chalice metadata
 app = Chalice(app_name='foursight-cgap')
 app.debug = True
 STAGE = os.environ.get('chalice_stage', 'dev')
-DEFAULT_ENV = 'cgap'
+DEFAULT_ENV = os.environ.get("ENV_NAME", "cgap-unknown")
 app_utils_obj = AppUtils()
 
+# When running 'chalice local' we do not get the same "/api" prefix as we see when deployed to AWS (Lambda).
+# So we set it explicitly here if your CHALICE_LOCAL environment variable is set; the point is to make
+# running locally as much as possible like running in AWS, at least as far is behavior is concerned;
+# but this does of course mean we're going through a slightly different code path for local and AWS.
+# Seems to be a known issue: https://github.com/aws/chalice/issues/838
```

Comment:
Does this change need to be ported to https://github.com/4dn-dcic/foursight/pull/501 ?

Reply:
Yes, should do that. It will also be taken care of - eventually - by moving the routes to foursight-core.